skype/skype-ps/skype/Get-CsInboundExemptNumberPattern.md

```diff
@@ -0,0 +1,121 @@
+---
+external help file: Microsoft.Rtc.Management.Hosted.dll-help.xml 
+online version: https://docs.microsoft.com/powershell/module/skype/get-csinboundexemptnumberpattern
+applicable: Skype for Business Online and Teams
+author: jenstr

```

Comment:
```suggestion
author: jenstrier

```

Reply:
Hello Jens, _author_ metadata should be a github username.

Reply:
Thanks! I've updated the files